themes/default/content/resources/walkthrough-of-building-container-based-aws-lambda-functions/index.md

```diff
@@ -0,0 +1,75 @@
+gated: false
+
+# The layout of the landing page.
+type: webinars
+
+external: false
+block_external_search_index: false
+
+# The url slug for the video landing page. If this is an external
+# video, use the external URL as the value here.
+url_slug: "walkthrough-of-building-container-based-aws-lambda-functions"
+
+# The content of the hero section.
+hero:
+    # The title text in the hero. This also serves as the pages H1.
+    title: "Walkthrough of building container-based AWS Lambda Functions"
+
+# Content for the left hand side section of the page.
+main:
+    # Video title.
+    title: "Walkthrough of building container-based AWS Lambda Functions"
+    # URL for embedding a URL for ungated videos.
+    youtube_url: "https://www.youtube.com/embed/4RC37yZJakM"
+    # Sortable date. The datetime Hugo will use to sort the videos in date order.
+    sortable_date: 2020-12-01T10:00:00-07:00
+    # Duration of the video.
+    duration: "7 minutes"
+    # Description of the video.
+    description: |
+        In this week's episode of Modern Infrastructure Wednesday, we go in-depth on the container-based AWS Lambda functions demo.
+
+        We walk through each line of code, showing how it works as well as what's in the `Dockerfile` to build the image.
+
+        Code for this episode available [here](https://github.com/pulumi/pulumitv/tree/master/modern-infrastructure-wednesday/2020-12-02).
+
+        Today's example is in TypeScript, but Pulumi makes it easy to stand up infrastructure in your favorite languages including JavaScript, Python, C#, and Go - saving time over legacy tools like CloudFormation and Hashicorp Terraform.
+
+    # The video presenters
+    presenters:
+        - name: Lee Zen
+          role: VP of Engineering, Pulumi
+
+# This section contains the transcript for a video. It is optional.
+transcript: |
+    Hello, and welcome to another episode of Modern Infrastructure Wednesday. Today, we're going to be going over the demo that it built for showing how to use container-based Lambda functions. And so we're going to walk through that code in depth. Since in the video, it was like kind of a quick take on just how to build an application and kind of show the demo of the application, but it never really walked through it in depth. And so we're going to do that in this video today. If we kind of start at the beginning, we import a couple of libraries. We're building a TypeScript application today Pulumi obviously does support Go, .NET as well as Python in addition to Node.js. So, but here we're using TypeScript. So I create this bucket you know, simply just call, you know, a new bucket. Here, we just name it bucket, but obviously that's just the, the name we're using within Pulumi. With Pulumi's auto naming capabilities these actually ended up being transformed into suffix names and the actual cloud provider in this case AWS. This next line here, we're using this convenience library, it'll be AWSX, this is a Crosswalk for AWS.
```

Comment:
```suggestion
    Hello, and welcome to another episode of Modern Infrastructure Wednesday. Today, we're going to be going over the demo that I built for showing how to use container-based Lambda functions. And so we're going to walk through that code in depth. Since in the video, it was like kind of a quick take on just how to build an application and kind of show the demo of the application, but it never really walked through it in depth. And so we're going to do that in this video today. If we kind of start at the beginning, we import a couple of libraries. We're building a TypeScript application today. Pulumi obviously does support Go, .NET as well as Python in addition to Node.js. So, but here we're using TypeScript. So I create this bucket you know, simply just call, you know, a new bucket. Here, we just name it bucket, but obviously that's just the, the name we're using within Pulumi. With Pulumi's auto naming capabilities these actually end up being transformed into suffix names and the actual cloud provider in this case AWS. This next line here, we're using this convenience library, AWSX, this is a Crosswalk for AWS.
```